cms/static/js/es2015/data/store.js

```diff
@@ -0,0 +1,12 @@
+import { createStore } from 'redux';
+import { syncCollections } from 'backbone-redux';
+
+const StudioStore = createStore(
+  () => {},
+);
+
+syncCollections({
+  TextbooksCollection: window.models.TextbookCollection,
```

Comment:
This is the money shot right here. We take our global reference to a Backbone collection and we map it up to our Redux store. This call creates a new reducer `TextbooksCollection` in the store, which remains synced to the Backbone collection. Apparently it also [adds a whole bunch of other things](https://github.com/redbooth/backbone-redux#how-to-use) like create action creators, which I haven't used, and which isn't especially well-documented. But there's no better documentation than the source code itself!

Reply:
This is awesome! I understand that this allows us to listen to the Backbone collection. I don't understand Backbone very well or how it's used in the legacy code, but could there be a situation in which our React components would need to somehow modify the collection? For example if you wanted to have a textbook manager that could add and remove textbooks. Would we go about that via the traditional action creator => reducer  => store route? Would those changes to collection in the Redux store appear in the actual Backbone collection?